apps/design-land/src/app/modal/modal-content/modal-content.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, OnInit } from '@angular/core';
+import { DaffModalService } from '@daffodil/design';
+
+@Component({
+  selector: 'design-land-modal-content',
+  templateUrl: './modal-content.component.html',
+})
+export class DesignLandModalContentComponent {
+  private modal: any;
+  constructor(private modalService: DaffModalService) {}
+
+  showModal() {
+    this.modal = this.modalService.open(DesignLandModalContentComponent);
+  }
+
+  hideModal() {
```

Comment:
maybe add an "x" to the modal content to demonstrate this? Or just remove this function

Reply:
Fixed in b75505d2cbbe48d71a604ee3a1eb1a5c597ebc24